pytests/tests/test_quiet.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2019 - 2023 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+
+import pytest
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_test(utils):
+    yield
+    pkg = utils.config["sglversion_pkgname"]
+    teardown_test(utils, pkg)
+
+
+def teardown_test(utils, pkg):
+    cmd = f"tdnf erase -qy {pkg}".split()
+    utils.run(cmd)
+
+
+def ensure_empty_ret(ret):
+    assert ret["retval"] == 0
+    assert not "\n".join(ret["stdout"])
+    stderr_l = []
+    for ele in ret["stderr"]:
+        if "NOKEY" not in ele:
+            stderr_l.append(ele)
+
+    assert not stderr_l
+
+
+err_str = "--assumeyes|-y or --assumeno is must when quiet is set"
+
+def ensure_non_empty_ret(ret):
+    assert ret["retval"] == 909
```

Comment:
This fails, I guess you meant `904`?

But also see my other comment - add a new code.